web/js/containers/animation-widget.js

```diff
@@ -139,12 +139,6 @@ class AnimationWidget extends React.Component {
     }
   }
 
-  static getDerivedStateFromProps(props, state) {
```

Comment:
Though a rare case, I think this would be needed if there are two consecutive tour steps with animations. 

Adding a location `key` to the animation widget could be a good replacement for `getDerivedStateFromProps`... similar to https://github.com/nasa-gibs/worldview/blob/master/web/js/app.js#L119

Reply:
Thanks, that change fixes tour related issues.